docs/technical_reference/property_models/seawater_ion_generic.rst

```diff
@@ -0,0 +1,87 @@
+Seawater Ion Generic Property Package
+=====================================
+
+This package implements property relationships for artificial seawater as provided in `Islam et al. (2021) <https://www.sciencedirect.com/science/article/abs/pii/S1383586621009412>`_.
+
+This artificial seawater property package:
+   * supports only 'H2O', 'Na_+', 'Ca_2+', 'Mg_2+', 'Cl\_-', 'SO4_2-'
+   * supports only liquid phase
+   * is formulated on a molar basis
+   * does not support dynamics
+
+Sets
+----
+.. csv-table::
+   :header: "Description", "Symbol", "Indices"
+
+   "Components", ":math:`j`", "['H2O', 'Na_+', 'Ca_2+', 'Mg_2+', 'Cl\_-', 'SO4_2-']"
+   "Phases", ":math:`p`", "['Liq']"
+
+State variables
+---------------
+.. csv-table::
+   :header: "Description", "Symbol", "Variable", "Index", "Units"
+
+   "Component molar flowrate", ":math:`N_j`", "flow_mol_phase_comp", "[p, j]", ":math:`\text{mole/s}`"
+   "Temperature", ":math:`T`", "temperature", "None", ":math:`\text{K}`"
+   "Pressure", ":math:`P`", "pressure", "None", ":math:`\text{Pa}`"
+
+Properties
+----------
+.. csv-table::
+   :header: "Description", "Symbol", "Variable", "Index", "Units"
+
+   "Component mole fraction", ":math:`y_j`", "mole_frac_phase_comp", "[p, j]", ":math:`\text{dimensionless}`"
+   "Component mole flowrate", ":math:`N_j`", "flow_mol_phase_comp", "[p, j]", ":math:`\text{mole/s}`"
+
+
+Relationships
+-------------
+.. csv-table::
+   :header: "Description", "Equation"
+
+   "Component mole flowrate", ":math:`N_j = \frac{M_j}{MW_j}`"
+   "Component mole fraction", ":math:`y_j = \frac{N_j}{\sum_{j} N_j}`"
+
```

Comment:
I don't think this is necessary because none of the other documentation have a note like this, but perhaps this is something we should add to all the documentation?

Reply:
If no property package has this, it is OK not inlcuding the note.